src/gui/editors/PianoRoll.cpp

```diff
@@ -171,6 +175,10 @@ PianoRoll::PianoRoll() :
 	m_oldNotesEditHeight( 100 ),
 	m_notesEditHeight( 100 ),
 	m_ppb( DEFAULT_PR_PPB ),
+	m_octaveHeight(s_keyLineHeight * KeysPerOctave),
+	m_whiteKeySmallHeight(round(s_keyLineHeight * 1.5)),
+	m_whiteKeyBigHeight(round(s_keyLineHeight * 2)),
+	m_blackKeyHeight(round(s_keyLineHeight * 1.3333)),
```

Comment:
I think it's also possible to use integer multiplications and divisions to save the computation power, but the difference will be negligible.

Reply:
Without `round()` I sometimes saw 1 pixel position errors - especially for black keys.

Reply:
I mean you can do it like `(s_keyLineHeight * 8 + 4) / 6`, though it looks unclear a bit.

Reply:
@PhysSong  So, should I change that? It's hard to decide (for me) is it a small optimization or readability of the code more important. Usually in such cases I choose the readability of the code. I removed unnecessary `round()` when multiplying by 2.